survos/bunny-bundle/1.5/config/packages/survos_bunny.yaml

```
@@ -0,0 +1,5 @@
+survos_bunny:
+    # this is generated automatically with bin/console bunny:config <api-key>
+    api_key: '%env(BUNNY_API_KEY)%'
+    # optionally set a default storage zone
+    storage_zone:
```

```suggestion
    #storage_zone:
```

And maybe add a hint at what you're expecting here like:

```suggestion
    #storage_zone: something
```